conf/application.conf.template

```diff
@@ -155,9 +155,9 @@ slick.dbs.default {
         driver = "org.postgresql.Driver"
         url = "jdbc:postgresql://localhost/ore"
         url = ${?JDBC_DATABASE_URL}
-        user = "root"
+        user = "spongeauth"
```

Comment:
leave the defaults, i don't know what this will break. the docker container & local users should set this however they want.

Reply:
This is Ore's connection to the database, it shouldn't use the same credentials as the spongeauth connection